app/src/main/res/values/strings.xml

```diff
@@ -64,17 +64,17 @@
     <string name="importOptionApplicationButton">Use external app</string>
 
     <string name="about">About</string>
-    <string name="app_copyright_fmt" translatable="false" tools:ignore="PluralsCandidate">Copyright 2019–<xliff:g>%d</xliff:g> Sylvia van Os.</string>
-    <string name="app_copyright_old">Based on Loyalty Card Keychain, copyright 2016–2020 Branden Archer.</string>
-    <string name="app_license">Copylefted libre software, licensed GPLv3+.</string>
+    <string name="app_copyright_fmt" tools:ignore="PluralsCandidate">Copyright © 2019–<xliff:g>%d</xliff:g> Sylvia van Os.</string>
+    <string name="app_copyright_old">Based on Loyalty Card Keychain\ncopyright © 2016–2020 Branden Archer.</string>
+    <string name="app_license">🄯 Copylefted libre software, licensed GPLv3+.</string>
     <string name="about_title_fmt">About <xliff:g id="app_name">%s</xliff:g></string>
-    <string name="debug_version_fmt">Version: <xliff:g id="version">%s</xliff:g></string>
+    <string name="debug_version_fmt"><big>Version: <xliff:g id="version">%s</xliff:g></big></string>
```

Comment:
Please don't add font size stuff to the translatables, these should be in code. It makes no sense for a translator to decide to make something bold or not.